src/__tests__/to-have-style.js

```diff
@@ -170,6 +200,24 @@ describe('.toHaveStyle', () => {
       })
     })
 
+    test("doesn't consider the unit if it isn't explicitly set ", () => {
+      const {queryByTestId} = render(`
+      <span data-testid="color-example" style="font-size: 12rem">Hello World</span>
+    `)
+      expect(queryByTestId('color-example')).toHaveStyle({
+        fontSize: 12
```

Comment:
Why would this match `12rem`? If I set `font-size: 12;` then this results in `12px`. This seems a bit confusing to me.

Also: This test only works in JSDOM. In an actual browser the value would be resolved to (depending on the base font size) to `240px`.

Reply:
You are right! This test case doesn't make any sense as it is going to work only in JSDOM. Btw, I didn't know that, thanks! I'm going to change that.

It isn't clear to me if you are only against this test case or the idea of the api.

Reply:
Just want to make sure that we're not documenting a test case that wouldn't work in a correct DOM environment.

Reply:
Follow up on this. It seems from the discussion above that there's still something to be addressed here before this is ready. Right?

Reply:
Is JSDOM actually returning `12rem` here has the computed style? Then all we have to do is add a comment explaining that this test depends on the user agent (since it will return the computed font-size in pixel which depends on the base font size for `rem`).

Reply:
I changed to `px` right after your first comment, do you think that is still worth to leave a note about that behavior?